### PR TITLE
Rewrite splash page

### PR DESCRIPTION
**Pull request by brianhlin**
Just a short blurb for the doc area home page, mainly to help direct admins and researchers to the proper locations. @osg-cat assigning this to you since reviewing actual content is in your wheelhouse.

The eventual idea is to replace the contents of https://twiki.opensciencegrid.org/bin/view/SoftwareTeam/WebHome with a link to this page or the [Install Overview](https://opensciencegrid.github.io/docs/install-overview/) page.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,42 +1,13 @@
-Installation Guides
-===================
+Open Science Grid Site Documentation
```
Comment (osg-cat) [inline docs/index.md]: OSG Site Administrator Documentation [would be my suggestion]

File: docs/index.md [hunk 2]
```diff
@@ -1,42 +1,13 @@
...
+====================================
```
Comment (osg-cat) [inline docs/index.md]: Use the # method for headings, IMO.

File: docs/index.md [hunk 3]
```diff
@@ -1,42 +1,13 @@
...
 
-The Open Science Grid consists primarily of a fabric of services at
-participating sites.
+!!! note
+    This is currently a work in progress as we migrate our TWiki documentation to GitHub, so please let us know of any issues that you encounter by [opening a ticket](https://ticket.opensciencegrid.org) or e-mailing [goc@opensciencegrid.org](goc@opensciencegrid.org) with a description of your issue.
```
Comment (osg-cat) [inline docs/index.md]: To what does the initial “This” refer? Almost always, the pronouns “this” and “that” should be followed by a noun phrase to indicate the referent; there are exceptions, but you should start with the noun and go from there.

Also, I would give more concrete suggestions than “with a description of your issue” — how about something like (as a new sentence), “Please identify errors, inconsistencies, or incorrect information in the documentation, and also suggest extra content that you feel would be helpful to include in the documentation.”

File: docs/index.md [hunk 4]
```diff
@@ -1,42 +1,13 @@
...
 
-One of the most common ways of participating in the OSG is to install
-one of our software services at your site and provide computational
-power opportunistically to the grid.
+Welcome to the home page of the Open Science Grid (OSG) Site documentation! The documentation contained here is designed for system adminstrators that provide resources to the OSG. If you're a new resource provider, the [Installation Overview](install-overview) is a good place to get started. Otherwise, you can find a list of our documents via the navigation and search bars to the left.
```
Comment (osg-cat) [inline docs/index.md]: Let’s aim to be consistent in our terminology. Are they system administrators or site administrators? Also, this whole paragraph is very chatty — can you tighten and focus it?

File: docs/index.md (hunk 2 above)
Comment (brianhlin) [reply docs/index.md]: I prefer the `====` and `----` method for h1 vs h2 headers, respectively. It's easy to spot and search for them in the source.

**Comment by brianhlin** [comment]
@osg-cat how's it look now?

**Comment by osg-cat** [comment]
OK.

**Pull request merged**